about.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>DRC Investment Portal | About</title>
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div class="page-loader" id="page-loader"></div>
+    <div class="wrapper" id="wrapper">
+      <section class="head">
+        <div class="social set">
+          <a href="#"><i class="fa fa-facebook-f social-icon"></i></a>
+          <a href="#"><i class="fa fa-twitter social-icon"></i></a>
+          <a href="#"><span class="social-icon">French</span></a>
+          <a href="#"><span class="social-icon">My Page</span></a>
+          <a href="#"><span class="social-icon">Logout</span></a>
+        </div>
+      </section>
+      <section class="top-nav">
+        <nav class="set">
+          <div class="nav-item1">
+            <a href="index.html"><img src="img/logo.png" alt="Logo"></a>
+          </div>
+          <div class="nav-item2">
+            <a href="about.html">About</a>
```

Comment:
### OPTIONAL
- [ ] Kindly change this link to `index.html`, this way it is possible to navigate back to the index.html page from the about page 👍🏿